marlowe-test/src/Spec/Marlowe/Semantics/Arbitrary.hs

```diff
@@ -441,14 +436,11 @@ randomValidatorHashes =
   , "a2bd7ddd1d11c7f4994fa7f41c2781c750525705f9c259f97cb27d0e"
   , "c5b4c54ec387ad8250b183a0d0d181617bb18bcf2eccc0f27fe7aa23"
   , "d877b83fee4a52bd72269ece77d78549fa64e111aa0e20cd4a1c471b"
-  , "e14025ab9bb3c48601d1893f86785145bcbe9e2e1856c16cfc0522"      -- NB: Too short for ledger.
-  , "e3351d2e500f17b91a74e89f3eaa66492193f4485c32e5ad606da83542"  -- NB: Too long for ledger.
   ]
 
 
 instance Arbitrary POSIXTime where
   arbitrary = POSIXTime <$> arbitraryInteger
-  shrink x = filter (< x) fibonaccis
```

Comment:
It is unnecessary to shrink integer values

Reply:
Why?

Reply:
Because one integer value is not structurally smaller than another (though I see QuickCheck does define rather elaborate shrinking logic for integral values, I can restore this.